src/components/FileUploader/index.tsx

```diff
@@ -107,15 +107,16 @@ export const FileUploader: FC<BoxProps & FileUploaderProps> = (p) => {
         <BodySm>{headerHelperText}</BodySm>
       </Box>
       <Box __css={styles.uploadArea} ref={dragAreaRef}>
-        <VStack spacing={4}>
-          <Image
-            maxW="80px"
-            src={useColorModeValue(fileUploadLight, fileUploadDark)}
-          />
+        <Image
+          maxW="80px"
+          mx="auto"
+          src={useColorModeValue(fileUploadLight, fileUploadDark)}
+        />
 
-          <BodyLg>
-            {isUploaded ? file?.name : "Drag and drop your file here"}
-          </BodyLg>
+        <BodyLg isTruncated mt="4" textAlign="center">
```

Comment:
I tested it out and it seems that the truncating does not work 🤔 

<img width="1523" alt="image" src="https://user-images.githubusercontent.com/40306834/232027904-61f446b5-b6cd-4d14-97da-f0fbdcb87fc9.png">

Reply:
Could you please double-check? It works form me in storybook and in Threshold dashboard.

![obraz](https://user-images.githubusercontent.com/57687279/234572918-ea380a01-f461-454d-87c6-404422491b51.png)

![obraz](https://user-images.githubusercontent.com/57687279/234573019-3673ce18-1c28-4261-9934-c34cab4754d8.png)